Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.1"
 
 [deps]
 DiffRules = "b552c78f-8df3-52c6-915a-8e097449b14b"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
Can't you just depend on AbstractFFTs.jl and write the dispatches using the high level functions?